medusa/processTV.py

```diff
@@ -489,54 +487,55 @@ def unRAR(path, rarFiles, force, result):  # pylint: disable=too-many-branches,t
     return unpacked_files
 
 
-def already_postprocessed(dirName, videofile, force, result):  # pylint: disable=unused-argument
+def already_postprocessed(dir_name, video_file, force, result):
     """
-    Check if we already post processed a file
+    Check if we already post processed a file.
 
-    :param dirName: Directory a file resides in
-    :param videofile: File name
+    :param dir_name: Directory a file resides in
+    :param video_file: File name
     :param force: Force checking when already checking (currently unused)
     :param result: True if file is already postprocessed, False if not
     :return:
     """
     if force:
         return False
 
-    # Avoid processing the same dir again if we use a process method <> move
+    file_path = os.path.join(dir_name, video_file)
+    file_size = os.path.getsize(file_path)
+
     main_db_con = db.DBConnection()
-    sql_result = main_db_con.select("SELECT release_name FROM tv_episodes WHERE release_name IN (?, ?) LIMIT 1", [dirName, videofile.rpartition('.')[0]])
-    if sql_result:
-        # result.output += logHelper(u"You're trying to post process a dir that's already been processed, skipping", logger.DEBUG)
+    # Try generic method first
+    episodes_result = main_db_con.select(
+        'SELECT file_size '
+        'FROM tv_episodes '
```

Comment:
No, for a few reasons:
- History can be deleted
- Download, replace, download same episode would fail
- We want to know what we currently have (that's why the size check) and not what we ever got (history)